index.html

```diff
@@ -2971,27 +2940,30 @@ <h3>Production</h3>
         <h3>Consumption</h3>
 
         <p>
-The <a>DID document</a> MUST be deserialized as a JSON document according to
-the <a href="#consumption">consumption rules for JSON</a>, with one additional
-requirement: The serialized <a>DID document</a> MUST include the
-<code>@context</code> member and be processed according to the rules below.
+All data structures expressed by a JSON-LD <a>representation</a> MUST be
+deserialized into the <a href="#data-model">data model</a> according to the JSON
+<a>representation</a> <a>consumption</a> rules as defined in <a
+href="#json"></a>.
         </p>
 
-        <dl>
-          <dt>@context</dt>
-          <dd>
-            <p>
-The value of the <code>@context</code> member conforms to the
-<a href="#production-0">JSON-LD Production Rules</a>. If more than one
-<a>URI</a> is provided, the <a>URIs</a> MUST be interpreted as an
-<a data-cite="INFRA#ordered-set">ordered set</a>.
-            </p>
-          </dd>
-        </dl>
+        <p>
+In addition to using the JSON <a>representation</a> <a>consumption</a> rules,
+JSON-LD consumption MUST add the representation-specific entries into a
+representation-specific <a href="#data-model">data model</a> according to the
```

Comment:
```suggestion
JSON-LD consumption MUST add the representation-specific entries into
the <a href="#data-model">data model</a> according to the
```

Reply:
@peacekeeper I've put this back and raised issue #664 to address this one. @OR13 has also raised #662 with a slightly different approach. I would prefer that we merge this PR as is (because it implements current consensus, even though none of us like it), and then we do a future PR to fix this once and for all once issues #662 and #664 are settled.